met-web/src/components/engagement/form/EngagementFormTabs/Settings/SendReport.tsx

```diff
@@ -1,22 +1,26 @@
-import React, { useContext, useEffect } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { Grid, FormGroup, FormControlLabel, Switch } from '@mui/material';
 import { MetHeader4, MetDescription, MetLabel } from 'components/common';
 import { EngagementTabsContext } from '../EngagementTabsContext';
 import { ActionContext } from '../../ActionContext';
 import { useAppDispatch } from 'hooks';
 import { openNotification } from 'services/notificationService/notificationSlice';
-import { EngagementSettingsContext } from './EngagementSettingsContext';
 
 const SendReport = () => {
-    const { settings, settingsLoading } = useContext(EngagementTabsContext);
+    const { settings, sendReport, setSendReport, settingsLoading } = useContext(EngagementTabsContext);
     const { savedEngagement } = useContext(ActionContext);
-    const { sendReport, setSendReport } = useContext(EngagementSettingsContext);
+    const [initialSendReportFlag, setInitialSendReportFlag] = useState(settings.send_report);
```

Comment:
Is there a reason why we're using two different syntaxes for sendReport? i.e. `sendReport` and `send_report`?

Reply:
Yes, it is related to the property coming from the DB. Similar is the case with is_internal.